### PR TITLE
feat: automatically add app for review in testflight

### PR DESCRIPTION
**Pull request by bonomat**
first step for https://github.com/get10101/meta/issues/443

### CODE REVIEW

**Comment by bonomat** [comment]
Note that we do have a problem: 
reviewing the app takes some time, if in the meantime the deployed coordinator changes with some breaking changes, the released reviewed app won't work. 

Given we don't guarantee anything on regtest, I think we can live with this.

File: .github/workflows/testflight-for-review.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        run: cargo install just --force
+
+      - name: Install fastlane
+        run: |
+          cd mobile/ios/fastlane
+          gem install bundler
+          bundle install
+          bundle info fastlane
+
+      - name: Put for beta testing app fro review
```
Comment (holzeis) [inline .github/workflows/testflight-for-review.yml:45]:
```suggestion
      - name: Put for beta testing app for review
```

**Review by holzeis — APPROVED** [review]
Approved, although the App Store approval is a bit cumbersome to coordinate with a coordinator upgrade.

What do you think of only supporting users on our test vm with the apk?

**Comment by bonomat** [comment]
> What do you think of only supporting users on our test vm with the apk?

Given this action shouldn't cost us anything I would just run it as well and just don't guarantee anything. We don't have breaking changes too often 🤞.

**Pull request merged**